src/online/online_player_profile.cpp

```diff
@@ -403,7 +403,8 @@ namespace Online
                     }
                     else if(to_notify.size() > 3)
                     {
-                        message = _("%d friends are now online.",
+                        message = _P("%d friends are now online.",
+                                     "%d friends are now online.",
```

Comment:
I'm not sure I understand, shouldn't the first one be singular? They are both plural so I don't quite see how this works. Also, this code is inside "if(to_notify.size() > 3)" so it's always plural. Unless there's something else going on I'm missing